README.md

```diff
@@ -425,3 +425,24 @@ var table = React.render(
 
 These can be useful if you want to roll your own filtering input field
 outside of Reactable.
+
+### Empty Data Sets
+
+If the table is initialized without any `<Tr>`s or with an empty array for
+`data`, it will display "No data available in table" text in the body
```

Comment:
As implemented, it will show the text from `defaultProps` for `noDataText`, although I don't feel strongly about that behavior and can remove it from default props.